.github/workflows/merge_workflow.yml

```diff
@@ -0,0 +1,103 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [main]
+    tags: ['*']
+  pull_request:
+    branches: [main]
+
+env:
+  SCALA_VERSION: 2.13.8
+  JAVA_VERSION: 11
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test-type: ['scala', 'python']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      if: matrix.test-type == 'scala'
+      uses: actions/setup-java@v2
+      with:
+        java-version: ${{ env.JAVA_VERSION }}
```

Comment:
are these env vars set? I don't see them. Same with the secrets you reference.

Reply:
No, I just have them set in the yml itself